README.md

```diff
@@ -112,6 +112,40 @@ jobs:
 ```
 The above workflow will apply policy files changes only in policies/** directory. For each directory, the action will first apply the definition and then assignments that have 'testRG' in their filename. This assignment field is especially useful for risk mitigation scenarios, where you first want to apply assignments corresponding to a specific environment like 'test'. 
 
+
+### Sample workflow to apply policies at Management Group scope
+
+```yaml
+# File: .github/workflows/workflow.yml
+
+on: push
+
+jobs:
+  apply-azure-policy:    
+    runs-on: ubuntu-latest
+    steps:
+    # Azure Login at management group scope requires allow-no-subscriptions to be set to true
+    - name: Login to Azure
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZURE_CREDENTIALS}}
+        allow-no-subscriptions: true
+
+    - name: Checkout
+      uses: actions/checkout@v2 
+
+    - name: Create or Update Azure Policies
+      uses: azure/manage-azure-policy@v0
+      with:      
+        paths:  |                
+          policies/**
+        
+```
+For deploying policies or initiatives at a management group level, the azure login action should have input `allow-no-subscriptions` set to true.
```

Comment:
definitons or initiatives?  Initiatives will fall under policies right?

Reply:
Not really. Here we are following the hierarchy in policy UI. Eg, Initiative definitions or Policy definitions